### PR TITLE
Bump SDK version; test updates

### PR DESCRIPTION
**Pull request by cwallsfdc**
Tests need more love.

### CODE REVIEW

File: middleware/lib/sfMiddleware.ts [hunk 1]
```diff
@@ -97,7 +95,12 @@ function createOrg(logger: Logger, reqContext: any, accessToken?: string): Org {
         throw new Error(message);
     }
 
-    const apiVersion = reqContext.apiVersion || process.env.FX_API_VERSION || Constants.CURRENT_API_VERSION;
+    const apiVersion = reqContext.apiVersion || process.env.FX_API_VERSION;
+    if (!apiVersion) {
```
Comment (cwallsfdc) [inline middleware/lib/sfMiddleware.ts:99]: We need an API version and cannot assume a version.

Comment (jabrown85) [reply middleware/lib/sfMiddleware.ts:99]: Where does `reqContext.apiVersion` come from? The payload or from the sf org?

Comment (cwallsfdc) [reply middleware/lib/sfMiddleware.ts:99]: Correct.

Comment (dhagberg-sf) [reply middleware/lib/sfMiddleware.ts:99]: nice.

**Review by dhagberg-sf — APPROVED** [review]
Thanks - tests have been neglected for past couple months.

**Pull request merged**